release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2874,6 +2874,68 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.10
+[id="ocp-4-17-10_{context}"]
+=== RHBA-2024:11522 - {product-title} {product-version}.10 bug fix, and security update advisory
+
+Issued: 02 January 2024
+
+{product-title} release {product-version}.10 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:11522[RHBA-2024:11522] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:11525[RHBA-2024:11525] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.10 --pullspecs
+----
+
+[id="ocp-4-17-10-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-17-10-node-tuning-operator_{context}"]
+===== Node Tuning Operator architecture detection
+
+The Node Tuning Operator can now properly select kernel arguments and management options for Intel and AMD CPUs. (link:https://issues.redhat.com/browse/OCPBUGS-43664[*OCPBUGS-43664*]) 
+
+[id="ocp-4-17-10-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when the webhook token authenticator was enabled and had the authorization type set to `None`, the {product-title} web console would consistently crash. With this release, a bug fix ensures that this configuration does not cause the {product-title} web console to crash. (link:https://issues.redhat.com/browse/OCPBUGS-46390[*OCPBUGS-46390*])
+
+// https://issues.redhat.com/browse/OCPBUGS-46082 marked as internal. Check closer to the release date if the Jira switched status.
```

Comment:
Ignoring for now and will recheck when advisories are in the SHIPPED_LIVE state.